README.md

```diff
@@ -1,281 +1,133 @@
+    ```
+    gds aws paas-$ACCOUNT-admin -- make $ENV upload-all-secrets
+    ```
+
+   **N.B** if you're deploying a short-lived environment (e.g. for penetration testing), the Github OAuth credentials will need to come from your personal `pass` store, and the path to it will be given by the `GITHUB_PASSWORD_STORE_DIR` env var
+    ```
+    gds aws paas-$ACCOUNT-admin -- make $ENV upload-all-secrets GITHUB_PASSWORD_STORE_DIR=/path/to/store
+    ```
+6. Wait for the pipeline to finish (this could take up to a couple of hours)
+7. The persistent Concourse is now created, and Concourse Lite can be killed by running the `self-terminate` pipeline.
+8. In [the Google API Console](https://console.cloud.google.com/apis/credentials), add the following URIs as authorised redirect URIs to the relevant OAuth 2.0 client configurations. This must be done manually, and cannot be automated. It is required for Google authentication.
+
+   The value of the $SYSTEM_DOMAIN placeholder here can be found under the `SYSTEM_DNS_ZONE_NAME` variable in the `Makefile`
+
+   N.B. All development environments fall under the "dev" name
+
+   | OAuth 2.0 Client Config       | URI                                                                       |
+   | ----------------------------- | ------------------------------------------------------------------------- |
+   | GOV.UK PaaS Operator - $ENV   | https://login.$SYSTEM_DOMAIN/login/callback/admin-google                  |
+   | GOV.UK PaaS Operator - $ENV   | https://uaa.$SYSTEM_DOMAIN/login/callback/admin-google                    |
+   | GOV.UK PaaS Operator - $ENV   | https://bosh-external.$SYSTEM_DOMAIN:8443/login/callback/admin-google     |
+   | GOV.UK PaaS Operator - $ENV   | https://bosh-uaa-external.$SYSTEM_DOMAIN:8443/login/callback/admin-google |
+   | GOV.UK PaaS - PaaS Admin $ENV | https://admin.$SYSTEM_DOMAIN/my-account/use-google-sso/callback           |
+   | GOV.UK PaaS $ENV Operator Grafana | https://grafana-1.$SYSTEM_DOMAIN |
+
+## Deploying the bootstrap for build environments
+1. Clone `paas-bootstrap` at the tip of the `main` branch
+2. Launch Concourse Lite.
+    ```
+    gds aws paas-$ACCOUNT-admin -- make $ENV build-concourse bootstrap
+    ```
+3. Log in to Concourse List at http://127.0.01:8080/login using the credentials shown in the output from step 2
+4. Run the `create-bosh-concourse` pipeline
+5. When the pipeline reaches the `check-for-secrets` job, it will pause for up to an hour (checking every 10 seconds) to allow secrets to be uploaded:
+    ```
+    gds aws paas-$ACCOUNT-admin -- make $ENV upload-all-secrets
+    ```
+   **N.B** if you're deploying a short-lived environment (e.g. for penetration testing), the GitHub OAuth credentials will need to come from your personal `pass` store, and the path to it will be given by the `GITHUB_PASSWORD_STORE_DIR` env var
```

Comment:
```suggestion
   Note: if you're deploying a short-lived environment (for example, for penetration testing), the GitHub OAuth credentials will need to come from your personal `pass` store, and the path to it will be given by the `GITHUB_PASSWORD_STORE_DIR` env var
```